src/main/java/ea/maps/Region.java

```diff
@@ -0,0 +1,34 @@
+package ea.maps;
+
+import ea.data.Color;
+
+public class Region {
```

Comment:
Change this to an enum and remove the state of isActive. All state needs to go in the GameState/State class